platform/darwin/src/MGLLight.h

```diff
@@ -0,0 +1,66 @@
+#import <CoreGraphics/CoreGraphics.h>
+
+#import "MGLFoundation.h"
+#import "MGLStyleValue.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+typedef NS_ENUM(NSUInteger, MGLLightAnchorType) {
+    MGLLightAnchorTypeMap,
+    MGLLightAnchorTypeViewport
+};
+
+MGL_EXPORT
+@interface MGLLight : NSObject
```

Comment:
This class any any other new public symbols will need documentation comments. The [style specification documentation](https://www.mapbox.com/mapbox-gl-js/style-spec/#light) can be a starting point, although we’ll need to tailor a few details to make sense in Objective-C and Swift.

Any new top-level symbols, such as MGLLight, should also be added to both platforms’ jazzy table of contents.